### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by penguineer**
This PR contains improvements to the documentation.

* [x] README in [standard-readme style](https://github.com/RichardLitt/standard-readme)
* [x] Specify the license (see #45)
* [x] Add [SPDX](https://spdx.org/) compliant license documentation
* [x] Adhere to guidelines of the [reuse initiative](https://reuse.software/)

### CODE REVIEW

**Comment by penguineer** [comment]
Which license is intended for this project? 
(A standard license's name would be cool ;) )

File: README.md [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
-### Build
-in the root of this project, run:
- mkdir build
- cd build
- cmake ..
- make 
- make install
-
-to build and install the program.
+Ideally, this should be split up into multiple daemons communicating over standardized protocols.
```
Comment (christf) [inline README.md:16]: not any more. l3roamd does just client presence and client roaming. The other mentioned functionality (manage distribution of prefixes) has been implemented in prefixd.

File: README.md [hunk 2]
```diff
@@ -42,9 +29,8 @@ In worst case two nodes may have to switch routes fast and repeatably due to a c
 to either node. The whole cycle should be
 These nodes may not be connected directly.
 
- 
-## [DB] Distributed Database
- 
+
+### [DB] Distributed Database
```
Comment (christf) [inline README.md:34]: this section should be removed. There is no distributed database. The functionality "does anyonne know this client" ist implemented by flooding the mesh.

File: README.md [hunk 3]
```diff
@@ -88,12 +74,13 @@ I.e. it will actively deprecate prefixes of clients it deems unreliably.
 This is likely to happen during roaming longer distances when a completely different set of uplinks should be used.
 As stated before, this will not break active connections.
 
-## IPv4?
+### IPv4?
```
Comment (christf) [inline README.md:78]: at this point I am uncertain whether this would work in an ipv4-only network.

File: README.md [hunk 4]
```diff
@@ -1,35 +1,22 @@
...
+
+> l3roamd, pronunced *ɛl θriː ɹoʊm diː*, is a core part of layer 3 mesh networks.
+
+l3roamd will will be built to work with [babeld](https://github.com/jech/babeld).
 
 Integrating tightly with mac80211 and netlink, l3roamd will be doing the following things:
  - monitor the presence of clients
  - allow network-wide search for IP-addresses
  - manage distribution of prefixes across the mesh (for router advertisements) [RA]
  - monitor babeld for duplicate host routes that this node also announces
```
Comment (christf) [inline README.md:14]: there is no dependency on babeld. This line should be dropped.

**Comment by penguineer** [comment]
I merely adapted the structure. Maybe remarks regarding the content should go to a distinct issue? Otherwise this PR will "explode".

**Comment by penguineer** [comment]
We assume the following license for all files lacking a specific annotation: https://opensource.org/licenses/BSD-2-Clause

**Comment by penguineer** [comment]
Seems that #45 ended up with https://spdx.org/licenses/BSD-2-Clause.html

**Pull request merged**